packages/loader/README.md

```diff
@@ -1,3 +1,12 @@
 # Fluid Loader
 
-For an overview of the Fluid Loader, please refer to the [@FluidFramework/container-loader package README](container-loader/README.md).
+This is about the "loader layer" of the FLuid Framework, not to be confused with the `Loader` class (which is part of this layer).
+
+The `loader` layer contains code which gets directly included into fluid applications.
+This this layer is kept minimal to reduce the the need to update the application when the Fluid Framework changes,
+as well as keep the size application small.
+Instead most of the code is placed in either the `runtime` or `driver` layers and dynamically loaded by the `loader`.
+
+The loading is initiated by loading a container, which involves the appropriate drivers and runtime for the container.
```

Comment:
Something about "The loading is initiated by loading a container" strikes me as odd but I don't know of a better way to say it. 🤔